src/theme/components/select.ts

```diff
@@ -24,7 +24,7 @@ const select = (labelPosition: string) => ({
   fontSize: "desktop.body.body2",
   minHeight: { base: defaultElementSizes.mobileFieldHeight, md: "auto" },
   paddingTop: "inset.narrow",
-  paddingEnd: "inset.extrawide",
+  paddingEnd: "2.5rem",
```

Comment:
@bigfishdesign13 we don't have a design token for this value?

Reply:
No. And we needed something in-between  what we do offer. `inset.wide` equates to `l` which is `32px` and `inset.extrawide` equate to `xl` which is `48px`. This needed `40px`.

Reply:
And do we need to update the changelog and Storybook doc version for this or do we feel like this is too small a change?